ocean_lib/assets/v4/test/test_asset.py

```diff
@@ -0,0 +1,218 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from ocean_lib.assets.v4.asset import V4Asset
+from ocean_lib.common.agreements.service_types import ServiceTypesV4
+from ocean_lib.services.v4.service import V4Service
+from tests.resources.ddo_helpers import (
+    get_key_from_v4_sample_ddo,
+    get_sample_v4_ddo,
+    get_sample_v4_ddo_with_compute_service,
+)
+
+ENCRYPTED_FILES_URLS = "0x044736da6dae39889ff570c34540f24e5e084f4e5bd81eff3691b729c2dd1465ae8292fc721e9d4b1f10f56ce12036c9d149a4dab454b0795bd3ef8b7722c6001e0becdad5caeb2005859642284ef6a546c7ed76f8b350480691f0f6c6dfdda6c1e4d50ee90e83ce3cb3ca0a1a5a2544e10daa6637893f4276bb8d7301eb35306ece50f61ca34dcab550b48181ec81673953d4eaa4b5f19a45c0e9db4cd9729696f16dd05e0edb460623c843a263291ebe757c1eb3435bb529cc19023e0f49db66ef781ca692655992ea2ca7351ac2882bf340c9d9cb523b0cbcd483731dc03f6251597856afa9a68a1e0da698cfc8e81824a69d92b108023666ee35de4a229ad7e1cfa9be9946db2d909735"
```

Comment:
- if using something like this, first please move it to some different file, where it does not compromise readability for an otherwise excellently-written class
- also make sure to leave an indication (a commeent) of whether this is an encryption of something, e.g. what should it decrypt to, or if it is just a placeholder etc.
- it looks a bit long to me. Is it a real encrypted files string taken from provider, or is it something else, more random? Maybe my eye is fooled by the GitHub interface but an encrypted files to be around 2-3 lines.

Reply:
I retrieved it from the DDO structure.